src/addrman.cpp

```diff
@@ -55,12 +55,10 @@ double CAddrInfo::GetChance(int64_t nNow) const
     double fChance = 1.0;
 
     int64_t nSinceLastSeen = nNow - nTime;
-    int64_t nSinceLastTry = nNow - nLastTry;
+    int64_t nSinceLastTry = std::max<int64_t>(nNow - nLastTry, 0);
 
     if (nSinceLastSeen < 0)
```

Comment:
@jtimon See the comment in the PR body:

> Please note that the nSinceLastSeen is intentionally skipped since nSinceLastSeen is unused and is being removed as part of #9532.

:-)